src/Api/Controller/SendConfirmationEmailController.php

```diff
@@ -43,17 +43,17 @@ class SendConfirmationEmailController implements ControllerInterface
     protected $url;
 
     /**
-     * @var TranslatorInterface
+     * @var Translator
      */
     protected $translator;
 
     /**
      * @param \Flarum\Settings\SettingsRepositoryInterface $settings
      * @param Mailer $mailer
      * @param UrlGenerator $url
-     * @param TranslatorInterface $translator
+     * @param Translator $translator
      */
-    public function __construct(SettingsRepositoryInterface $settings, Mailer $mailer, UrlGenerator $url, TranslatorInterface $translator)
+    public function __construct(SettingsRepositoryInterface $settings, Mailer $mailer, UrlGenerator $url, Translator $translator)
```

Comment:
Can we please undo these changes and stick with injecting the Symfony TranslatorInterface. I think we want that to be our primary, implementation of the Illuminate contract is secondary just so the validator still works.